### PR TITLE
Update .github files

### PR DESCRIPTION
**Pull request by sergiught**
## Description

In this PR we are updating the .github templates for the SDK. 

## Testing

- [x] This change adds test coverage for new/changed/fixed functionality

## Checklist

- [x] I have read and agreed to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
- [x] I have read the [Auth0 General Contribution Guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md).
- [x] I have reviewed my own code beforehand.
- [x] I have added documentation for new/changed functionality in this PR.
- [x] All active GitHub checks for tests, formatting, and security are passing.
- [x] The correct base branch is being used, if not `master`.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: 🚀 Feature Request
+description: Suggest an idea or a feature for this SDK
+labels: [ "feature request" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/feature_request.yml:9]:
```suggestion
      value: Thanks for taking the time to fill out this feature request!
```
There's no need for the pipe if it's a single line.

File: .github/ISSUE_TEMPLATE/report_a_bug.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+name: 🐛 Report a bug
+description: Have you found a bug or issue? Create a bug report for this SDK
+labels: [ "bug" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/report_a_bug.yml:9]:
```suggestion
      value: Thanks for taking the time to fill out this bug report!
```
Same

File: .github/ISSUE_TEMPLATE/report_a_bug.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this bug report!
+
+  - type: markdown
+    attributes:
+      value: |
+        **Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/responsible-disclosure-policy) details the procedure for disclosing security issues.
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/report_a_bug.yml:13]:
```suggestion
      value: **Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/responsible-disclosure-policy) details the procedure for disclosing security issues.
```
Same as above.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,53 +1,42 @@
...
-Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. 
-If this library has unit and/or integration testing, tests should be added for new functionality and 
-existing tests should complete without errors.
+<!--
+Describe how this can be tested by reviewers. Be specific about anything not tested and why. Include any manual steps for testing end-to-end, or for testing functionality not covered by unit tests.
 -->
 
-- [ ] This change adds test coverage for new/changed/fixed functionality
-
+### 📝 Checklist
```
Comment (Widcket) [inline .github/PULL_REQUEST_TEMPLATE.md:35]: I'd suggest moving this to the top, right above "Changes", so it's the first thing people see, and they don't waste time filling in the previous fields only to find out they still can't raise the PR.

File: .github/ISSUE_TEMPLATE/report_a_bug.yml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      label: Description
+      description: Provide a clear and concise description of the issue.
+    validations:
+      required: true
+
+  - type: textarea
+    id: expectation
+    attributes:
+      label: Expectation
+      description: Tell us about the behavior you expected to see.
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/report_a_bug.yml:44]: Nit: not sure about the value of having this as a separate field.

**Comment by Widcket** [comment]
GH reports that the CODEOWERS is not valid.

Screen Shot 2022-10-27 at 10 29 26

File: .github/ISSUE_TEMPLATE/report_a_bug.yml (hunk 5 above)
Comment (sergiught) [reply .github/ISSUE_TEMPLATE/report_a_bug.yml]: I removed it, I agree with you it's a but superfluous.

File: .github/ISSUE_TEMPLATE/report_a_bug.yml [hunk 6]
```diff
@@ -37,14 +37,6 @@ body:
     validations:
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/report_a_bug.yml:37]: Maybe include `, including the behavior you expected to see.` in the description ob the above field? So both fields become integrated.

Comment (sergiught) [reply .github/ISSUE_TEMPLATE/report_a_bug.yml:37]: Done

**Review by Widcket — APPROVED** [review]
LGTM

**Pull request merged**